src/piecetype.cpp

```diff
@@ -65,6 +65,21 @@ auto Piece::fromWind(Wind w) -> Piece {
   }
 }
 
+auto Piece::formPiece(Piece::Type suit, uint8_t number) -> Piece {
+  auto valueAppend = number;
+  if (number == 1 || number == 9) {
+    valueAppend |= TERMINAL_BIT;
+  } else if (number == 0) {
```

Comment:
I really don't like this. Can we have an additional red five parameter?

Reply:
It is a part of the community standard for 0 to represent red five, this should be discussed further